### PR TITLE
Updates for newest version of the PLSS

### PR DESCRIPTION
**Pull request by mheagin**
Updates to the new version, update dates, downloads and Point_AGRC info

### CODE REVIEW

File: data/cadastre/plss/index.html [hunk 1]
```diff
@@ -144,16 +152,14 @@ <h5>Related Resources</h5>
...
-    <p>PLSSPoint_AGRC is a point dataset that represents the AGRC schema of the GCDB point layer. We are currently (Oct. 2019) redesigning it to reflect Points Submitted, Points incorporated into the fabric (Control), Points that have older Tie Sheets that are monument records with no X,Y coordinates. It also has a link to a copy of the tie sheet or the county website for those. The layer PLSSPoint_GCDB on the SGID and available for download, is the original source for this layer. This layer is used in the PLSS Corner Management App.</p>
-    <h5>Notes for PLSSPoint_AGRC feature class</h5>
+    <img class="inline-text-left outline" src="{% link images/plsspoint_agrc.jpg %}" alt="PLSS Point AGRC" />
+    <p><strong>The new release of Points_AGRC data layer is now created from the PLSSPoint_GCDB and a “Control” points feature class in the Fabric. The “Control” points are collected tie sheets points submitted to the AGRC that have been used to adjust and improve the PLSS Fabric.</h5>
```
Comment (steveoh) [inline data/cadastre/plss/index.html:159]: This strong needs to be closed or removed.

File: data/cadastre/plss/index.html [hunk 2]
```diff
@@ -144,16 +152,14 @@ <h5>Related Resources</h5>
...
     <ul>
```
Comment (steveoh) [inline data/cadastre/plss/index.html:154]: This ul should be removed

**Review by steveoh — CHANGES_REQUESTED** [review]
The text from agrc point is fouled up from the strong an ul and needs fixing before merging. I can take care of it if you don't get to it first.

File: data/cadastre/plss/index.html (hunk 2 above)
Comment (mheagin) [reply data/cadastre/plss/index.html:154]: sorry about the mess, go ahead and remove it. Thanks Not sure how I can edit it after I do a pull.

**Review by steveoh — APPROVED** [review]

**Pull request merged**